code.js

```diff
@@ -1,44 +1,37 @@
-function permutationSort(a, length, tests) {
-
-    /*if(a.length == length)
-    {
-        tests++
-        test = true
-        // Compare each value to each other value to ensure the array is sorted
-        for(i=0; i < a.length && test == true; i++)
+function permutationSort(a) {
+    function bruteForce(a, start)
+    { 
+        for(i = 0; i < array.length; i++)
         {
-            for(j = i; j < a.length && test == true; j++)
+            swapIndices(i, i+1, a)
```

Comment:
Here you probably want to make sure that you can get back to what it was before, otherwise you're swapping elements in the swapped list, which won't generate all permutations.